modules/@apostrophecms/admin-bar/ui/apos/components/TheAposContextModeAndSettings.vue

```diff
@@ -30,27 +30,10 @@
       :key="'switchToPreviewMode'"
       class="apos-admin-bar__control-set__group"
     >
-      <AposButton
-        v-if="context._id && !hasCustomUi"
-        class="apos-admin-bar__context-button"
-        label="Page Settings" :tooltip="{
```

Comment:
Where did this go?

Reply:
It moved into this https://github.com/apostrophecms/apostrophe/pull/3271/files#diff-f0cd06d001d5414ddb9a2fd35e6410566bb6af241d4ff83a5f25f2b4751fb587L53